packages/now-cli/src/commands/dev/dev.ts

```diff
@@ -17,8 +17,6 @@ export default async function dev(
   args: string[],
   output: Output
 ) {
-  output.dim(`Now CLI ${pkg.version} dev (beta) — https://zeit.co/feedback`);
```

Comment:
Do we not want feedback for `now dev` anymore?

Reply:
We moved it here: https://github.com/zeit/now/blob/6a1b1b7173c6f2d3ac2bbc423024b693803d57f2/packages/now-cli/src/index.js#L164-L172

I've updated it to always show the feedback prompt when the command is `dev` (as opposed to only show it for `canary`).